### PR TITLE
Add about and works

### PR DESCRIPTION
**Pull request by Berkay-akbas**
Sections Works and About me added.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -39,6 +39,117 @@ <h2>I'am a Software Developer</h2>
       </ul>
     </footer>
   </section>
+  <section class="works">
+    <Header>
+      <p>My Recent Works</p>
```
Comment (ShahierNashaat) [inline index.html:44]: - [ ] Kindly be noticed that the title `My Recent Works` should be inside an `hx` tag.

File: index.html [hunk 2]
```diff
@@ -39,6 +39,117 @@ <h2>I'am a Software Developer</h2>
...
+    <p><img class="about_icon3" src="images/Rectangle57.png" alt="icon">Skills</p>
+    <ul class="Skills">
+      <li>Codekit</li>
+      <li>Github</li>
+      <li>Codepen</li>
+      <li>Gitlab</li>
+      <li>Terminal</li>
+    </ul>
+
+  </section>
```
Comment (ShahierNashaat) [inline index.html:152]: - [ ] Kindly be noticed that you should put all of the about content in one `section` tag.
- [ ] Kindly be noticed that the technology categories ("Languages", "Frameworks" and "Skills") are in a `ul`, each of them in a `li`.

File: styles.css [hunk 3]
```diff
@@ -95,10 +97,237 @@ body {
...
+  font-size: 36px;
+  color: #fff;
+  list-style-type: none;
+  font-weight: 700;
+}
+
+.set ul {
+  line-height: 40px;
+  margin-top: -20px;
+}
```
Comment (ShahierNashaat) [inline styles.css:315]: - [ ] It is better to add for this `ul` padding: 0 so when it shows in the center of the page.

**Review by ShahierNashaat — CHANGES_REQUESTED** [review]
Hi @Berkay-akbas , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (Berkay-akbas) [reply index.html:152]: Hello, thank you for your feedback and I'm sorry I don't understand what you mean by the second line.  Could you elaborate, please?

Comment (ShahierNashaat) [reply index.html:152]: The "Languages", "Frameworks" and "Skills" all have to be inside of a one `ul` tag and each one of them should be inside a `li` tag and inside that `li` tag create a `ul` tag like you have created and add inside it the `li` tags like this.

```
<ul>
<li>
<p><img class="about_icon1" src="images/Rectangle55.png" alt="icon">Language</p>
    <ul class="language">
      <li>Javascipt</li>
      <li>Ruby</li>
      <li>Html</li>
      <li>Css</li>
    </ul>
</li>
<li>
    <p><img class="about_icon2" src="images/Rectangle56.png" alt="icon">Frameworks</p>
    <ul class="Frameworks">
      <li>Bootstrap</li>
      <li>Ruby on Rails</li>
      <li>RSPec</li>
      <li>CapyBara</li>
      <li>Selenium</li>
    </ul>
</li>
<li>
    <p><img class="about_icon3" src="images/Rectangle57.png" alt="icon">Skills</p>
    <ul class="Skills">
      <li>Codekit</li>
      <li>Github</li>
      <li>Codepen</li>
      <li>Gitlab</li>
      <li>Terminal</li>
    </ul>
</li>
</ul>
```

Comment (Berkay-akbas) [reply index.html:152]: Much appreciated!

File: styles.css [hunk 4]
```diff
@@ -95,10 +97,240 @@ body {
...
+  color: #fff;
+  list-style-type: none;
+  line-height: 42.67px;
+  font-weight: 100;
+}
+
+.set img {
+  margin-right: 15px;
+}
+
```
Comment (mukhammadrizooff) [inline styles.css:331]: - [ ] [RQUIRED] Please make changes! The technology categories ("Languages", "Frameworks" and "Skills") should true use the margins (top, bottom, left, right) defined in the design. 

image

File: styles.css [hunk 5]
```diff
@@ -95,10 +97,240 @@ body {
...
+.card li {
+  padding: 8px;
+  font-size: 15px;
+  font-family: Inter, sans-serif;
+  line-height: 20px;
+  color: #0fff;
+  background-color: rgba(107, 107, 107, 0.5);
+}
```
Comment (mukhammadrizooff) [inline styles.css:227]: - [ ] [OPTIONAL] Good job! Please use colors in [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1).

image

**Review by mukhammadrizooff — CHANGES_REQUESTED** [review]
Hi @Berkay-akbas , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
- All linters passed 💯 
- Correct use of Gitflow 💯 
- Descriptive messages 💯 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by vic778 — APPROVED** [review]
## Hi @Berkay-akbas  , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Charlie Brown

## Highlights

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**Pull request merged**